index.html

```diff
@@ -52,6 +61,234 @@ <h2 class="sub-title">I am a Softwaer Developer</h2>
+    <div class="card-pro-text">A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</div>
+    <div class="card-pro-buttons">
+        <ul class="pro-buttons">
+            <li><a href="#">html</a></li>
+            <li><a href="#">css</a></li>
+            <li><a href="#">javascript</a></li>
+            
+        </ul>
+    </div>
+    <div class="card-last-button">See Project</div>
+
+   </section>
+   <!--professional part start 5-->
+   <section class="card2">
+    <div class="card-title">Profesional Art<br> Printing Data</div>
+    <div class="card-pro-text">A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</div>
+    <div class="card-pro-buttons">
+        <ul class="pro-buttons">
+            <li><a href="#">html</a></li>
+            <li><a href="#">css</a></li>
+            <li><a href="#">javascript</a></li>
+            
+        </ul>
+    </div>
+    <div class="card-last-button">See Project</div>
+
+   </section>
+   <!--professional part start 6-->
+   <section class="card2">
+    <div class="card-title">Profesional Art<br> Printing Data</div>
+    <div class="card-pro-text">A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</div>
+    <div class="card-pro-buttons">
+        <ul class="pro-buttons">
+            <li><a href="#">html</a></li>
+            <li><a href="#">css</a></li>
+            <li><a href="#">javascript</a></li>
+            
+        </ul>
+    </div>
+    <div class="card-last-button">See Project</div>
+
+   </section>
+   <!--about section start-->
+   <section class="card-3">
+    <div class="card-about-title">About me</div>
+    <div class="card-about-text">Hello I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</div>
+    <div class="card-about-button">Get My Resume</div>
+    <div class="about-img"><img src="img/Vector44.png" alt=""></div>
+    <!--skill part start-->
+    <div class="skill">
+       
+        <div class="card-skill">
+            
+            <h3 class="skill-img ab"> <img src="img/skill55.png" alt=""> Language</h3>
+            <ul>
+                <li class="skillname sn">
+                    <a href="#">Codekit</a>
+                </li>
+                <li class="skillname sn">
+                    <a href="#">Github</a>
+                </li>
+                <li class="skillname sn">
+                    <a href="#">Codepen</a>
+                </li>
+                <li class="skillname sn">
+                    <a href="#">Gitlap</a>
+                </li>
+                <li class="skillname sn">
+                    <a href="#">Terminal</a>
+                </li>
+            </ul>
+        </div>
```

Comment:
- [ ] Please make sure that the technology categories ("Languages", "Frameworks" and "Skills") are in a `<ul>`, each of them in a `<li>` for better semantic practice. I know you can do that 💪
- [ ] Also please ensure that the technologies in each category(e.g., "JavasScript", "HTML" and "CSS") are also a list `<ul>` and each of them a `<li>`. Great job for already doing that. You just ned to change a bit here 👍👌